azure-pipelines.yml

```diff
@@ -28,34 +29,61 @@ jobs:
   - powershell: .\build.ps1
     displayName: Build 
     env:
-      SIGNCLIENT_SECRET: $(SignClientSecret)
-      SIGNCLIENT_USER: $(SignClientUser)
-      ArtifactDirectory: $(System.DefaultWorkingDirectory)\artifacts
+       SIGNCLIENT_SECRET: $(SignClientSecret)
+       SIGNCLIENT_USER: $(SignClientUser)
+       CODECOV_TOKEN: $(CODECOV_TOKEN)
+       ArtifactDirectory: $(System.DefaultWorkingDirectory)\Artifacts
```

Comment:
ArtifactsDirectory?

Reply:
That variable isn't used but I'm going to swap over to using these soon so this file will be dramatically changing as well https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops